rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -124,25 +122,23 @@ rmw_create_client(
   subscriberParam.topic.topicKind = NO_KEY;
   subscriberParam.topic.topicDataType = response_type_name;
   subscriberParam.historyMemoryPolicy = PREALLOCATED_WITH_REALLOC_MEMORY_MODE;
-  rcutils_ret_t ret = _assign_partitions_to_attributes(
-    service_name, ros_service_response_prefix,
-    qos_policies->avoid_ros_namespace_conventions, &subscriberParam);
-  if (ret != RCUTILS_RET_OK) {
-    // error msg already set
-    goto fail;
+  if (!qos_policies->avoid_ros_namespace_conventions){
```

Comment:
Style: Indentation and spacing seems to not be matching the guidelines. You can find style violations by running the tests of the package locally

Reply:
Thanks for the catch. I've updated it in this commit https://github.com/ros2/rmw_fastrtps/pull/192/commits/0a516ea6f5528e7ce38a7ae40379cd778e89ff33